modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2ServiceJWTGrantTestCase.java

```diff
@@ -43,108 +45,128 @@
-    private final String COUNTRY_LOCAL_CLAIM_URI = "http://wso2.org/claims/country";
-    private final String EMAIL_OIDC_CLAIM = "email";
-    private final String EMAIL_CLAIM_VALUE = "email@email.com";
-    private final String EMAIL_LOCAL_CLAIM_URI = "http://wso2.org/claims/emailaddress";
     private String refreshToken;
-    String openIdScope = "openid";
+    private String applicationId;
+    private String userId;
+    private String idpId;
+    private String countryClaimId;
 
     @BeforeClass
     public void setup() throws Exception {
 
         super.init(TestUserMode.SUPER_TENANT_ADMIN);
         changeISConfiguration("jwt_token_issuer_enabled.toml");
         super.init(TestUserMode.SUPER_TENANT_ADMIN);
-        OAuthConsumerAppDTO appDto = createApplication(createApplicationWithJWTGrantType(), SERVICE_PROVIDER_NAME);
-        consumerKey = appDto.getOauthConsumerKey();
-        consumerSecret = appDto.getOauthConsumerSecret();
-        userManagementClient = new UserManagementClient(backendURL, sessionCookie);
-        oauthAdminClient = new OauthAdminClient(backendURL, sessionCookie);
-        userProfileMgtServiceClient = new UserProfileMgtServiceClient(backendURL, sessionCookie);
-        identityProviderMgtServiceClient = new IdentityProviderMgtServiceClient(sessionCookie, backendURL);
-        addNewUserWithClaims();
-        claimMetadataManagementServiceClient = new ClaimMetadataManagementServiceClient(backendURL, sessionCookie);
-
-        changeCountryOIDDialect();
-
-        ExternalClaimDTO externalClaimDTO = new ExternalClaimDTO();
-        externalClaimDTO
-                .setExternalClaimDialectURI(OAuth2ServiceAuthCodeGrantOpenIdRequestObjectTestCase.OIDC_CLAIM_DIALECT);
-        externalClaimDTO.setMappedLocalClaimURI(COUNTRY_LOCAL_CLAIM_URI);
-        externalClaimDTO.setExternalClaimURI(COUNTRY_NEW_OIDC_CLAIM);
-        claimMetadataManagementServiceClient.addExternalClaim(externalClaimDTO);
-        String[] openidValue = new String[1];
-        openidValue[0] = COUNTRY_NEW_OIDC_CLAIM;
-        oauthAdminClient.updateScope(openIdScope, openidValue, null);
+
+        ApplicationResponseModel application = createApplicationWithJWTGrantType();
+        applicationId = application.getId();
+        OpenIDConnectConfiguration oidcConfig = getOIDCInboundDetailsOfApplication(applicationId);
+        consumerKey = oidcConfig.getClientId();
+        consumerSecret = oidcConfig.getClientSecret();
+
+        scim2RestClient = new SCIM2RestClient(serverURL, tenantInfo);
+        claimManagementRestClient =  new ClaimManagementRestClient(serverURL, tenantInfo);
+        oidcScopeMgtRestClient =  new OIDCScopeMgtRestClient(serverURL, tenantInfo);
+        idpMgtRestClient = new IdpMgtRestClient(serverURL, tenantInfo);
+
+        addAdminUser();
+        changeCountryOIDCDialect();
     }
 
     @AfterClass(alwaysRun = true)
     public void atEnd() throws Exception {
+        deleteApp(applicationId);
```

Comment:
Shall we add the missing new line here.

Reply:
Fixed with https://github.com/wso2/product-is/pull/20096